### DISCUSSION TITLE
Not seeing requests in dev tools network tab

### DISCUSSION THREAD

**Discussion by Theqwertypusher** [usebruno/bruno#1945] [Q&A] [answered]
How exactly can I see requests in the dev tools network tab? I am able to make successful requests but the dev tools does not capture it. Previously, I used `v1.4.0` which did not work and then I upgraded to the latest `1.12.3` and it still does not capture requests. Any assistance is greatly appreciated!

![image](https://github.com/usebruno/bruno/assets/36286598/f6c2bba5-81cc-40fc-956e-6245c28662c5)

**Comment by GarlicPunk** [comment]
The timeline tab shows the request, headers submitted, however if you're running pre-requests scripts or functions that make additional calls these don't show up there. 
I use HTTP-Toolkit with Bruno to capture all the network traffic.

**Comment by quotschmacher** [comment]
@GarlicPunk but if I got the Developer Tools included, why should I use another tool? 

I was wondering, too, why the requests do not appear there.

**Comment by sanjai0py** [comment, marked as answer]
Kindly review the comments regarding this issue at the following: https://github.com/usebruno/bruno/issues/983
